frontend/src/routes/FacilityBooking/CreateBooking/index.tsx

```diff
@@ -172,20 +178,14 @@ export default function CreateBooking() {
         <LoadingSpin />
       ) : (
         <Form onSubmit={onSubmit}>
-          <Controller
-            control={control}
-            render={({ onChange, value }) => (
-              <InputField
-                hasError={value === ''}
-                title="Event Name"
-                placeholder="Event Name"
-                value={value}
-                onChange={onChange}
-              />
-            )}
-            defaultValue=""
-            rules={{ required: true }}
-            name="eventName"
+          <InputField
+            name={'eventName'}
+            title="Event Name"
+            placeholder="Event Name"
+            required
+            register={register}
+            setValue={setValue}
+            errors={errors.eventName}
           />
           {errors.eventName && <p>{errors.eventName?.message}</p>}
```

Comment:
is this necessary since the error is handled in the input field?

Reply:
OH i was thinking how come there was extra <p> there. Yeah I just removed it.

Reply:
[14b381d](https://github.com/rhdevs/rhapp/pull/637/commits/14b381d54773b7719a78c8cd1e8ab56f3ff5cda3)